.github/workflows/build-test-coverage.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           distribution: ${{ matrix.java-distribution }}
           cache: 'gradle'
       - name: Validate Gradle wrapper
-        uses: gradle/wrapper-validation-action@v1
+        uses: gradle/actions/wrapper-validation@v3
```

Comment:
Not raised by Dependabot but there were a few deprecation warnings in our pipelines